.rubocop.yml

```diff
@@ -2,15 +2,15 @@ require: rubocop-rails
 
 AllCops:
   Exclude:
-    - 'vendor/**/*'
-    - 'db/**/*'
-    - 'bin/**/*'
-    - 'Gemfile'
-    - 'Gemfile.lock'
-    - 'Rakefile'
-    - 'Guardfile'
-    - 'node_modules/**/*'
-    - 'config/**/*'
+    - "vendor/**/*"
```

Comment:
Why do we change to use double quote?

Reply:
Oh it done by my VSCode. will rever it @hoangmirs

Reply:
Reverted in https://github.com/nimblehq/rails-templates/pull/198/commits/ce7fa211ada4aeeb2881ffea2d61c64d6c9342fc